lazer/contracts/evm/src/PythLazer.sol

```diff
@@ -5,7 +5,9 @@ import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 
 contract PythLazer is OwnableUpgradeable, UUPSUpgradeable {
-    TrustedSignerInfo[2] public trustedSigners;
+    TrustedSignerInfo[2] private deprecatedTrustedSigners;
```

Comment:
Can't you just extend this one to become like 100, then add verification_fee after it?

Reply:
Ah that's a good idea. I'll change to doing it that way.